src/transformStateWithClones.js

```diff
@@ -7,7 +7,32 @@
  * @return {Object[]}
  */
 function transformStateWithClones(state, actions) {
-  // write code here
+  const eachAction = [];
```

Comment:
[NAMING] - The variable `eachAction` does not clearly describe its purpose. Consider renaming it to `stateHistory` or `statesAfterActions` to better reflect that it contains the history of states after each action.